addons/controllers/suite_test.go

```diff
@@ -275,6 +283,39 @@ var _ = BeforeSuite(func(done Done) {
 	err = testutil.CreateResources(f, cfg, dynamicClient)
 	Expect(err).ToNot(HaveOccurred())
 
+	labelMatch, _ := labels.NewRequirement("webhook-cert", selection.Equals, []string{"self-managed"})
+	labelSelector := labels.NewSelector()
+	labelSelector = labelSelector.Add(*labelMatch)
+
+	secret, err := webhooks.InstallNewCertificates(ctx, k8sConfig, certPath, keyPath, webhookScrtName, addonNamespace, webhookServiceName, "webhook-cert=self-managed")
```

Comment:
Will this line of code affect the test case in `certificates_test.go`?

Reply:
I made each test self-contained, so this should not affect the certificate_test.go. The test cases in certificate_test.go independently setup the certificates when run.